test/parallel/test-process-uid-gid.js

```diff
@@ -74,7 +74,8 @@ const oldgid = process.getgid();
 try {
   process.setgid('nobody');
 } catch (err) {
-  if (err.message !== 'setgid group id does not exist') {
```

Comment:
> This change adds exception handling for the error code received when process.setgid('nobody') is called on Ubuntu, where the standard 'nobody' group from UNIX systems is named 'nogroup'.

I think a bit more investigation is required, here. You can see in the code that it already handles non-existend of group `nobody`, it falls through to group `nogroup`, which you describe as the "standard 'nobody' group from UNIX systems". So, whatever is failing might be something other than that.  The `message` is strange below, too, because if you look in lib/internal/errors.js, you'll see `E('ERR_UNKNOWN_CREDENTIAL', '%s identifier does not exist: %s', Error);`, and the message does not match the format of the error code. I suspect the actual test bug may be that the message should be `'Group identifier does not exist'`.